candig/server/cli/server.py

```diff
@@ -23,9 +20,9 @@ def __init__(self, app, options=None):
 
     def load_config(self):
         config = dict(
-            [(key, value) for key, value in self.options.iteritems()
+            [(key, value) for key, value in self.options.items()
              if key in self.cfg.settings and value is not None])
-        for key, value in config.iteritems():
+        for key, value in config.items():
```

Comment:
or just
for key in config:
        self.cfg.set(key.lower(), config[key])
￼
￼

Reply:
Iterating through the pair seems fine, which eliminates the need of a dict look up?

Reply:
okay